addon/components/ember-tfoot/template.hbs

```diff
@@ -4,9 +4,14 @@
     rowIndex=rowIndex
     selectedRows=selectedRows
     columns=columns
+
+    onSelect=onSelect
+    selectMode=selectMode
     selectRow=selectRow
     toggleRowCollapse=toggleRowCollapse
 
+    metaCache=_rowMetaCache
```

Comment:
It seems like `_rowMetaCache` should not be underscored and "private" if it is passed to other components.